Practicals/prac_09/silver_service_taxi_test.py

```diff
@@ -0,0 +1,21 @@
+"""
+CP1404 Prac 9 - Silver Service Taxi test
+Test for the Silver Service Taxi class
+"""
+
+from prac_09.silver_service_taxi import SilverServiceTaxi
+
+
+def main():
+    """Client test code for a taxi"""
+
+    fuel = 200
```

Comment:
These values should be passed in directly as an object class, don't need to put them on a variable for testing.